locales/fil.json

```diff
@@ -112,8 +143,8 @@
     "aboutDetailes": "Ang Inji ay isang mobile app na maaaring gamitin bilang digital wallet para mag-imbak ng mga kredensyal. Pinapayagan din nito ang pag-verify ng pagkakakilanlan sa parehong offline at online na mga mode, sa anumang lugar at oras.",
     "forMoreDetailes": "Para sa karagdagang detalye",
     "clickHere": "Pindutin dito",
-    "version": "Bersyon:",
-    "tuvaliVersion": "Tuvali-bersyon"
+    "version": "Bersyon: 0.4.1",
```

Comment:
Version should not be hardcode, it should be extracted from package.json

Reply:
please do check for other languages files as well

Reply:
Removed version. tuvali-version in all the locals of AboutInji obj. They are further no need.